Multisig/UI/Transaction/TransactionListViewController/QueuedTransactionsViewController.swift

```diff
@@ -47,6 +47,15 @@ class QueuedTransactionsViewController: TransactionListViewController {
         date.timeAgo()
     }
 
+    override func shouldHighlight(transaction: SCGModels.TxSummary) -> Bool {
+        switch transaction.txInfo {
+        case .rejection(_):
+            return true
```

Comment:
The `transaction.txInfo == .rejection` doesn't work, does it?

Reply:
Nope unfortunately